components/cdc/src/endpoint.rs

```diff
@@ -2226,6 +2240,22 @@ mod tests {
         );
     }
 
+    #[test]
+    fn test_raw_causal_ts_flush() {
+        let sleep_interval = Duration::from_secs(1);
+        let cfg = CdcConfig {
+            min_ts_interval: ReadableDuration(sleep_interval),
+            ..Default::default()
+        };
+        let ts_provider = Arc::new(causal_ts::tests::TestProvider::default());
+        let start_ts = ts_provider.get_ts().unwrap();
+        let _suite =
+            mock_endpoint_with_ts_provider(&cfg, None, ApiVersion::V2, Some(ts_provider.clone()));
+        std::thread::sleep(sleep_interval * 2);
```

Comment:
How about test by schedule `Task::MinTS` as the above test case ? The sleep & wait will not be stable in a busy test infra.